src/Header/Header.jsx

```diff
@@ -144,10 +144,22 @@ Header.propTypes = {
 	enabled: PropTypes.bool,
 	search: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
 	cookie: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
-	auth: PropTypes.shape({
-		provider: PropTypes.string,
-		environment: PropTypes.string
-	}),
+	auth: PropTypes.oneOfType([
```

Comment:
I think we don't need to reproduce the full shape here. I think we can just use `PropTypes.object` (similar to what we do for search/cookie props and let the Account component define its prop types in full.